build_library/grub_install.sh

```diff
@@ -140,6 +140,10 @@ info "Generating ${GRUB_DIR}/load.cfg"
 ESP_FSID=$(sudo grub-probe -t fs_uuid -d "${LOOP_DEV}p1")
 sudo_clobber "${ESP_DIR}/${GRUB_DIR}/load.cfg" <<EOF
 search.fs_uuid ${ESP_FSID} root \$root
```

Comment:
So if you boot an ISO on a system that has a disk installed from the same build this fs_uuid is search is going to find it. When booting from ISO fs_uuid should not get called.